tests/node-tests/strip-data-test-properties-plugin-test.js

```diff
@@ -3,12 +3,12 @@ let assert = require('assert');
 
 let babel7 = require('@babel/core');
 
-let StripDataTestPropertiesPlugin6 = require('../strip-data-test-properties-plugin6');
+let StripDataTestPropertiesPlugin6 = require('../../ember-test-selectors/strip-data-test-properties-plugin6');
```

Comment:
we shouldn't be doing relative references like this across package boundaries. We need to add a `workspace:*` dependency to node-tests and then import it with package paths 👍